plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -104,19 +143,46 @@ def resource_template=(value)
     # dynamically fill out the templates and store the result
     def store_resource_template
       add_pod_disruption_budget
+      add_env_config_map
```

Comment:
prefer something like `env_config_map = add_env_config_map` and then use `if env_config_map` below to cut down on extra methods/interactions